src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -416,28 +415,135 @@ def _check_private_endpoint(cmd, registry_name, vnet_of_private_endpoint):  # py
     else:
         raise CLIError('DNS routing verification failed')
 
+# Validate inputs for --repository and --image
+def validate_repository_and_image(cmd, registry_name, repository_name, image):
```

Comment:
do we want to allow cx to use username and password to auth? do we want to allow optional rg param to avoid armcache?

Reply:
Added resource-group name to avoid armcache.